### PR TITLE
[MIG] Migrated web_ir_actions_act_window_message to v10.0

### PR DESCRIPTION
**Pull request by JonathanStein**
First attempt to migrate web_ir_actions_act_window_message to v10.0

### CODE REVIEW

**Comment by hbrunn** [comment]
please use [the guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0) to move commits from older branches into this one

File: web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js [hunk 1]
```diff
@@ -36,13 +36,13 @@ openerp.web_ir_actions_act_window_message = function(instance)
...
-                        'this': this,
-                        'action': action,
-                    }))
+                    $content: jQuery(instance.web.qweb.render(
+                        'web_ir_actions_act_window_message',
+                        {
+                            'this': this,
+                            'action': action,
+                        }))
+                }
```
Comment (hbrunn) [inline web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js:45]: all of this file needs to be migrated to v9 api

**Comment by pedrobaeza** [comment]
There's also an in progress migration to 9.0 in #605 that both can check for finding common solutions.

**Comment by benwillig** [comment]
@JonathanStein Are you still working on this PR?

**Comment by JonathanStein** [comment]
@benwillig I'm waiting for #611 (out of laziness - and because I think there's a point in keeping the versions as close as possible).

**Comment by sbidoul** [comment]
@JonathanStein #640 is ready for you to review and test. Closing this one then.

**Pull request closed without merging**